Source/JavaScriptCore/llint/LowLevelInterpreter.asm

```diff
@@ -2300,6 +2301,24 @@ end, dispatchAfterRegularCall)
 commonCallOp(op_construct, OpConstruct, prepareForRegularCall, invokeForRegularCall, prepareForSlowRegularCall, macro (getu, metadata)
 end, dispatchAfterRegularCall)
 
+commonCallOp(op_super_construct, OpSuperConstruct, prepareForRegularCall, invokeForRegularCall, prepareForSlowRegularCall, macro (getu, metadata)
+    if JSVALUE64
+        getu(m_argv, t1)
+        lshifti 3, t1
+        negp t1
+        addp cfr, t1
```

Comment:
Confused here. What's inside the current `t1` now? Is it the CFR for the the super call frame?

Reply:
Discussed offline. `m_argv` is offset for shifting the next cfr. So, at this point, `t1` is the cfr for the super call frame.